lib/src/pipeline/aggregation/exponential_histogram.rs

```diff
@@ -0,0 +1,343 @@
+use std::{
+    cmp::min,
+    f64::consts::{E, LN_2, LOG2_E},
+    sync::atomic::Ordering,
+};
+
+use crate::{pipeline::AbsorbDistribution, types::Distribution};
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct ExponentialHistogram {
+    scale: u8,
+    max_bucket_index: u16,
+    positive_buckets: Vec<usize>,
+    negative_buckets: Vec<usize>,
+}
+
+impl ExponentialHistogram {
+    pub fn new(scale: u8) -> Self {
+        Self::new_with_max_buckets(scale, 160)
```

Comment:
`160` is fixed, will there by support to make this configurable?

Reply:
Look to line 22 to see that this is configurable now

Reply:
Great, thank you!